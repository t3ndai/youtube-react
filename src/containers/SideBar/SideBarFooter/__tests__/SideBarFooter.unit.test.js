import React from "react";
import { shallow } from "enzyme";
import { SideBarFooter } from "../SideBarFooter";

test("it renders SideBarFooter", () => {
  const wrapper = shallow(<SideBarFooter />);
  expect(wrapper).toMatchSnapshot();
});
