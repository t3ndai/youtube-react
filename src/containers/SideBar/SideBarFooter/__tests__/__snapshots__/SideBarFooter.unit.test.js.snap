// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders SideBarFooter 1`] = `
<Fragment>
  <div
    className="footer-block"
  >
    <div>
      About Press Copyright
    </div>
    <div>
      Creators Advertise
    </div>
    <div>
      DEvelopers
    </div>
    <div>
      Legal
    </div>
  </div>
  <div
    className="footer-block"
  >
    <div>
      Terms Privacy
    </div>
    <div>
      Policy & Safety
    </div>
    <div>
      Test new features
    </div>
  </div>
  <div
    className="footer-block"
  >
    <div>
       All prices include VAT 
    </div>
  </div>
  <div
    className="footer-block"
  >
    <div>
       
      Â© Productioncoder.com - A Youtube clone for educational purporses
    </div>
  </div>
</Fragment>
`;
